src/withAxiom.ts

```diff
@@ -52,7 +52,7 @@ export function withAxiomNextConfig(nextConfig: NextConfig): NextConfig {
   };
 }
 
-export type AxiomRequest = (NextRequest | Request) & { log: Logger };
+export type AxiomRequest = NextRequest & { log: Logger };
```

Comment:
Did this change recently in Next.js? I think the `Request` was there for a reason (functions maybe?)

Reply:
No, next.js has changed that, but the way we did this makes the functions always assume its Request not NextRequest, dunno what's the explanation. according to issue #137 I thought using NextRequest would probably be fine for all functions.

![Screenshot 2023-08-21 at 11 47 02 AM](https://github.com/axiomhq/next-axiom/assets/49557793/12b7db51-e55e-46fd-8411-0be85e20f0a9)
example: If I try to use request.nextUrl it shows error, but it should work because its part of NextRequest.

Reply:
Alright!